build/Makefile

```diff
@@ -33,6 +33,7 @@ CWRP = "./src/wrapper.c"
 # Configure sqlite for out use-case
 SQLFLG = -DSQLITE_DQS=0 -DSQLITE_THREADSAFE=0 -DSQLITE_LIKE_DOESNT_MATCH_BLOBS\
          -DSQLITE_DEFAULT_FOREIGN_KEYS=1 -DSQLITE_TEMP_STORE=2\
+         -DSQLITE_ENABLE_FTS5\
```

Comment:
Can we add the option at the end rather than in the middle and also add a note to the comment below why we’re adding it?

Otherwise this looks good to go!

Reply:
I did that originally, then realized the options are in alphabetical order.

Reply:
I broke the options onto separate lines, alphabetized them, and fixed/added comments for any missing ones.